### PR TITLE
[Feature] Package crux

### PR DESCRIPTION
**Pull request by jpfeuffer**
Includes https://github.com/OpenMS/THIRDPARTY/pull/36

### CODE REVIEW

**Comment by jpfeuffer** [comment]
Are the crux tests finished @hroest?  Did I include the wrong version? It says the pin_out option is missing

**Comment by hroest** [comment]
Crux tests work here, but I was compiling from master and not from the latest release. They had some issues with their mzIdentML file format which should be fixed in the next release

**Comment by jpfeuffer** [comment]
Hmm did you maybe forget to push the right test.ini? Because the parameters in the ini do not really fit to the adapter.

**Comment by hroest** [comment]
and it is possible that it only works with the current git version as the crux guys did fix a few bugs that I hit while testing.

**Comment by jpfeuffer** [comment]
Seems to work now. I think the latest "release" was from a git snapshot anyway: Crux 3.1.8b78546

**Comment by hroest** [comment]
great, seems like this release is pretty recent:

```
commit 8b78546b823bf71adec32996bb255893c1a2bc6f
Author: Kaipo Tamura <kaipot@u.washington.edu>
Date:   Tue Jan 9 13:11:07 2018 -0800

    MzIdentMLWriter performance improvements
    Code reviewed by Charles

```

File: src/tests/topp/THIRDPARTY/third_party_tests.cmake [hunk 1]
```diff
@@ -136,7 +136,7 @@ endif()
 #------------------------------------------------------------------------------
 if (NOT (${CRUX_BINARY} STREQUAL "CRUX_BINARY-NOTFOUND"))
   ### NOT needs to be added after the binarys have been included
-  add_test("TOPP_CruxAdapter_1" ${TOPP_BIN_PATH}/CruxAdapter -test -ini ${DATA_DIR_TOPP}/THIRDPARTY/CruxAdapter_1.ini -database ${DATA_DIR_TOPP}/THIRDPARTY/proteins.fasta -in ${DATA_DIR_TOPP}/THIRDPARTY/spectra_comet.mzML -out CruxAdapter_1_out1.tmp -pin_out CruxAdapter_1_out2.tmp.csv -crux_executable "${CRUX_BINARY}" -run_percolator false)
+  add_test("TOPP_CruxAdapter_1" ${TOPP_BIN_PATH}/CruxAdapter -test -ini ${DATA_DIR_TOPP}/THIRDPARTY/CruxAdapter_1.ini -database ${DATA_DIR_TOPP}/THIRDPARTY/proteins.fasta -in ${DATA_DIR_TOPP}/THIRDPARTY/spectra_comet.mzML -out CruxAdapter_1_out1.tmp -crux_executable "${CRUX_BINARY}" -run_percolator false)
```
Comment (hroest) [inline src/tests/topp/THIRDPARTY/third_party_tests.cmake]: :+1: that seems to have been an older version of the adapter: d613d639bc34daa9f61b17aaa1aae39281ac4819

**Pull request merged**